implement_me.py

```diff
@@ -0,0 +1,115 @@
+from dateutil import parser
+from elasticsearch.helpers import bulk
+from elastic_handler import get_elastic_client
+
+INDEX_NAME = 'entry-index'
+ENTRY_TYPE = 'ip_entry'
+MAX_DOCUMENTS = 100000
+BULK_SIZE = 1000
+
+es = get_elastic_client()
+
+
+def clear_index():
+    es.indices.delete(index=INDEX_NAME, ignore=[400, 404])
+    init_mapping()
+
+
+def init_mapping():
+    index_mapping = """{
+        "mappings": {
+            "ip_entry":
+                {
+                  "properties": {
+                    "ip": { 
+                      "type":     "text",
+                      "fielddata": true
```

Comment:
This one is True by default (I think).

Reply:
It's not, this is the whole reason for this mapping...